test/Microsoft.AspNet.Mvc.Xml.Test/XmlDataContractSerializerOutputFormatterTest.cs

```diff
@@ -272,7 +272,7 @@ public void DefaultConstructor_ExpectedWriterSettings_Created()
             var body = outputFormatterContext.HttpContext.Response.Body;
             body.Position = 0;
 
-            var content = new StreamReader(body).ReadToEnd();
+            var content = new StreamReader(body, Encoding.Unicode).ReadToEnd();
```

Comment:
This change is fine because before the usage of custom stream writer, we used to [write out the BOM](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Common/Encodings.cs#L20) for utf-16 encoding and because of this i believe StreamReader was able to figure out that the content was utf-16 without the requirement of explicitly supplying the encoding...but since the custom stream writer avoids writing totally, the StreamReaders requires supplying the encoding...